toke.c

```diff
@@ -4470,76 +4482,112 @@ S_intuit_more(pTHX_ char *s, char *e)
     {
         return TRUE;
     }
-    else if (*s == '$') {
-
-        /* First char is dollar; lean very slightly to it being a subscript */
+    else if (*s == '$') { /* If the first char is dollar; lean very slightly to
+                             it being a subscript */
         weight = -1;
     }
-    else {
-        /* Otherwise, lean a little more towards it being a character class */
+    else { /* Otherwise, lean a little more towards it being a character class.
+            */
```

Comment:
Interesting, I much prefer the layout of these comments before the change - when I'm cleaning up gnarly code I'd tend to make exactly the opposite edit (except for the useless blank line at 4474).

Reply:
The new PR changes this a bit; and it just seems natural to me to do it similarly in this kind of context